Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM rustlang/rust@sha256:5af55c68b21232886d8d9bd35563b8a2ac0f71952369fb71346a51b331acd0d4 as rust_builder
+FROM rustlang/rust:nightly as rust_builder
```

Comment:
let's pin the image hash of whatever latest nightly is

Reply:
done

Reply:
done